README.md

```diff
@@ -8,16 +8,19 @@ Repository for automatically publishing the below crates every Tuesday.
 - [ra-ap_rustc_macros][ra-ap_rustc_macros-crate] [![ra-ap_rustc_macros version][ra-ap_rustc_macros-version-badge]][ra-ap_rustc_macros-crate]
 - [ra-ap_rustc_parse_format][ra-ap_rustc_parse_format-crate] [![ra-ap_rustc_parse_format version][ra-ap_rustc_parse_format-version-badge]][ra-ap_rustc_parse_format-crate]
 - [ra-ap_rustc_abi][ra-ap_rustc_abi-crate] [![ra-ap_rustc_abi version][ra-ap_rustc_abi-version-badge]][ra-ap_rustc_abi-crate]
+- [ra-ap_rustc_pattern_analysis][ra-ap_rustc_pattern_analysis-crate] [![ra-ap_rustc_pattern_analysis version][ra-ap_rustc_pattern_analysis-version-badge]][ra-ap_rustc_pattern_analysis-crate]
```

Comment:
Future archaeologists will see badge Markdown snippets and think they were ritualistic incantations to summon demons or gods.

Reply:
And they won't be entirely wrong :D